airflow/www/templates/appbuilder/flash.html

```diff
@@ -41,10 +41,12 @@
 
   {% if regular_alerts %}
     {% for category, m in regular_alerts %}
-      <div class="alert alert-{{ category if category else 'info' }}">
-        <button type="button" class="close" data-dismiss="alert">&times;</button>
-        {{ m }}
-      </div>
+      {% if not ('login' in request.path and 'access is denied' in m.lower()) %}
```

Comment:
Hmmm, don't feel great about this. But not sure I have a better idea.

Let's at least add a (jinja) comment  `{# #}` saying why we are doing this.

Better than just `'login' in request.path` is 

```suggestion
      {% if not (request.path == appbuilder.get_url_for_login and 'access is denied' in m.lower()) %}
```